src/google-maps/google-map/google-map.ts

```diff
@@ -245,6 +253,7 @@ export class GoogleMap implements OnChanges, OnInit, OnDestroy {
             'https://developers.google.com/maps/documentation/javascript/' +
             'tutorial#Loading_the_Maps_API');
       }
+      googleMapsWindow.gm_authFailure = () => this.authFailure.emit();
```

Comment:
I think that this will break if there are multiple maps on the page since each one will overwrite the auth failure callback on the previous map. We should do something like in the `youtube-player`: https://github.com/angular/components/blob/master/src/youtube-player/youtube-player.ts#L224

Reply:
Done